stacks/upload-api-stack.js

```diff
@@ -66,6 +66,7 @@ export function UploadApiStack({ stack, app }) {
     customDomain,
     defaults: {
       function: {
+        timeout: '30 seconds',
```

Comment:
Seems to max out at ~14s, setting to 30s to be safe.

<img width="1406" alt="Screenshot 2024-05-02 at 15 13 10" src="https://github.com/w3s-project/w3infra/assets/152863/03eb2ea0-360e-44d3-9327-ff227d534389">